### PR TITLE
Add README.md for quote verification

### PR DESCRIPTION
**Pull request by nick-mobilecoin**
Add readme with a high level summary of the quote verification process

This provides a more condensed version of the logic in the referenced Intel docs

Note: I've been heads down in this for a while so if something needs clarity or better description please let me know

The [rendered link](https://github.com/mobilecoinfoundation/sgx/tree/feature/quote_verification_readme/untrusted/quote_verify#signature-chain)

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #18** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/18" target="_blank">Graphite</a> 
    * **PR #19** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/19" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/19?utm_source=stack-comment).

File: untrusted/quote_verify/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+The chain includes the root CA, however since trust must first be established the accepted root CA is hardcoded in the
+crate, see (TODO add a link when available).
+
+The signature chain is documented
+in [Intel SGX PCK Certificate and Certificate Revocation List Profile Specification][2]. The chain is a hierarchy of
+X.509 V3 certificates.
+
+## Revocation Lists
+
+The Certificate Revocation Lists (CRL) needs to be retried from Intel
```
[the remark this answers is not in the document]
Comment (nick-mobilecoin) [reply untrusted/quote_verify/README.md]: good catch!

File: untrusted/quote_verify/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+1. In live runs, the Provisioning Key Certificate(PCK) has 1 TCB entry available in the certificate extensions. The
+   entry appears to be the highest. Is this enough? Will multiples be provided if there are multiples?
+2. Call back to Intel, https://api.trustedservices.intel.com/sgx/certification/v3/tcb?fmspc=00906ED50000. This requires
+   getting the `fmspc` out of the PCK certificate extensions.
+
+# References
+- https://download.01.org/intel-sgx/latest/dcap-latest/linux/docs/Intel_SGX_ECDSA_QuoteLibReference_DCAP_API.pdf
+- https://download.01.org/intel-sgx/latest/dcap-latest/linux/docs/SGX_PCK_Certificate_CRL_Spec-1.4.pdf
+
+<!-- Repeated here and in references for ease of linking --/>
```
[the remark this answers is not in the document]
Comment (nick-mobilecoin) [reply untrusted/quote_verify/README.md]: Nice spot on that, removing it seems to work now, thanks!

**Review by remoun — APPROVED** [review]
LGTM

**Pull request merged**